src/Crocodealer/Command/Sync.hs

```diff
@@ -0,0 +1,58 @@
+module Crocodealer.Command.Sync
+       ( sync
+       ) where
+
+import Network.HTTP.Simple (getResponseBody, getResponseStatus, httpBS, parseRequest)
+import Network.HTTP.Types.Status (Status (..))
+
+import Crocodealer.ColorTerminal (errorMessage, infoMessage, warningMessage)
+import Crocodealer.Core.GitHub (FileName (..), RepoName (..), UserName (..))
+
+
+data FileStatus
+    = UpToDate
+    | Outdated
+    | FileDoesNotExist
+
+sync :: UserName -> RepoName -> FileName -> [RepoName] -> IO ()
+sync userName templateRepoName fileName repos = do
+    templateFile <- fetchGitHubFile userName templateRepoName fileName
+    case templateFile of
+      Nothing -> printFileDiff userName templateRepoName FileDoesNotExist
+      Just f -> do
+        putTextLn $ unFileName fileName
+        forM_ repos $ \repo -> do
+            repoFile <- fetchGitHubFile userName repo fileName
+            case repoFile of
+              Nothing -> printFileDiff userName repo FileDoesNotExist
+              Just r -> do
+                let fileDiff = calculateDiff f r
+                printFileDiff userName repo fileDiff
+
+fetchGitHubFile :: UserName -> RepoName -> FileName -> IO (Maybe ByteString)
+fetchGitHubFile userName repoName fileName = do
+    let url = "https://raw.githubusercontent.com/" <> unUserName userName <> "/" <> unRepoName repoName <> "/master/" <> unFileName fileName
+    req <- parseRequest $ toString url
+    resp <- httpBS req
+
+    pure $ case getResponseStatus resp of
+        Status 200 _ -> Just $ getResponseBody resp
+        _            -> Nothing
+
+
+printFileDiff :: UserName -> RepoName -> FileStatus -> IO ()
+printFileDiff (UserName user) (RepoName repo) = \case
+    UpToDate -> infoMessage $ userRepo <> "Up-to-date"
+    Outdated -> warningMessage $ userRepo <> "Outdated"
+    FileDoesNotExist -> errorMessage $ userRepo <> "Doesn't have file"
+  where
+    userRepo = user <> "/" <> repo <> ": "
+
+calculateDiff :: ByteString -> ByteString -> FileStatus
+calculateDiff templateFile repoFile =
+    if checkSame templateFile repoFile
+    then UpToDate
+    else Outdated
+
+checkSame :: ByteString -> ByteString -> Bool
+checkSame templateFile repoFile = templateFile == repoFile
```

Comment:
Since `checkSame` functin became so simple, there's no need in it, you can simply write

```haskell
if templateFile == repoFile
then ...
else ...
```